_summaries/goodfellow2013maxout_2.md

```diff
@@ -1,19 +1,31 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: Maxout Networks
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: specfazhou
+score: 5
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+* What is the core idea? <br/>
+This paper comes up with a new function -- maxout function. Using Maxout function together with dropout techniques can classify images with lowest error rate on several well-known images dataset at that time. 
+
+* How is it realized (technically)? <br/>
+the input is a **d**-dim vertical vector, **X=(x_1,x_2,...x_d)^{t}**) with outputs **m**-dim vertical vector **h = (h_1,h_2,...,h_m)^{t}** in the next layer. Choose an integer **k**, and what the maxout function does is equivalent to adding another layer in between with **k** x **m** units, which forms a matrix **Z = (z_{i,j}**) where **0<i<k+1, 0<j<m+1**. Then, the **j**-th column of **Z** is **W(: , :, j)X**, where **W** is a **k** x **d** x **m** tensor with elements are parameters, and **h_j** is the maximum element in the **j**-th column of **Z**.      
```

Comment:
The detailed technical description is nice, but it feels a bit too dense with minute details.  Perhaps we can include a higher-level description of what it does.  Something like: "Each output neuron of a maxout layer is computed as the maximum of a single fully-connected layer.  Thus, the maxout layer can be seen as a concatenation of multiple fully-connected layers with "max" aggregation function."

Also, isn't the last **h_j** supposed to be **h_i**?

Reply:
I think the last h_j is correct. Thanks for your advice about high-level description though, since the summary requires us to write technical description about maxout function, maybe I just don't change this time.